test/functional/test_runner.py

```diff
@@ -115,6 +115,7 @@
     'feature_taproot.py',
     'rpc_signer.py',
     'wallet_signer.py --descriptors',
+    'wallet_signer.py --descriptors --invalid',
```

Comment:
Let's not fill this file up with too many options. `wallet_signer.py` can test both scenario's without needing an argument.

Reply:
done in https://github.com/bitcoin/bitcoin/pull/23628/commits/a5f53c36942bf82c056380df6c15adf0b12190f2